internal/appsec/dyngo/instrumentation/httpsec/tags.go

```diff
@@ -98,122 +104,12 @@ func NormalizeHTTPHeaders(headers map[string][]string) (normalized map[string]st
 	return normalized
 }
 
-// ippref returns the IP network from an IP address string s. If not possible, it returns nil.
-func ippref(s string) *instrumentation.NetaddrIPPrefix {
-	if prefix, err := instrumentation.NetaddrParseIPPrefix(s); err == nil {
-		return &prefix
-	}
-	return nil
-}
-
 // ClientIPTags returns the resulting Datadog span tags `http.client_ip`
 // containing the client IP and `network.client.ip` containing the remote IP.
 // The tags are present only if a valid ip address has been returned by
 // ClientIP().
-func ClientIPTags(hdrs map[string][]string, hasCanonicalMIMEHeaderKeys bool, remoteAddr string) (tags map[string]string, clientIP instrumentation.NetaddrIP) {
-	remoteIP, clientIP := ClientIP(hdrs, hasCanonicalMIMEHeaderKeys, remoteAddr)
-
-	remoteIPValid := remoteIP.IsValid()
-	clientIPValid := clientIP.IsValid()
-	if !remoteIPValid && !clientIPValid {
-		return nil, instrumentation.NetaddrIP{}
-	}
-
-	tags = make(map[string]string, 2)
-	if remoteIPValid {
-		tags["network.client.ip"] = remoteIP.String()
-	}
-	if clientIPValid {
-		tags[ext.HTTPClientIP] = clientIP.String()
-	}
-
+func ClientIPTags(headers map[string][]string, hasCanonicalMIMEHeaderKeys bool, remoteAddr string) (tags map[string]string, clientIP netip.Addr) {
```

Comment:
Can we rename this one too ? 🙏 😁

Reply:
Good catch, I forgot this function was here too.
Renamed in [5a91f1e](https://github.com/DataDog/dd-trace-go/pull/1846/commits/5a91f1e739ec9e9fd526a91558f25936c3ef5426)